### PR TITLE
init

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/makeRobotAccountant.js [hunk 1]
```diff
@@ -6,7 +6,23 @@
...
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return (firstNum) => {
+    counter++;
+
+    let num = firstNum;
+
+    return (secondNum) => {
```
Comment (MagicBox15) [inline src/makeRobotAccountant.js:16]:
```suggestion
  return (a) => (b) => {
    counter++;
    //and add some checking
    return a + b; 
```
I think you can replace with shorter construction

**Review by MagicBox15 — APPROVED** [review]
GJ!

**Review by AlinaMatuschak — CHANGES_REQUESTED** [review]
fix @MagicBox15 comment

**Review by AlinaMatuschak — APPROVED** [review]
🤘

**Pull request closed without merging**